src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -104,6 +106,52 @@ final class HomeViewController: UIViewController {
 		updateBackgroundColor()
 	}
 
+	/// This method sets up a background fetch alert, and presents it, if needed.
+	/// Check the `createBackgroundFetchAlert` method for more information.
+	func setupBackgroundFetchAlert() {
+		guard let alert = createBackgroundFetchAlert(
+			status: UIApplication.shared.backgroundRefreshStatus,
+			inLowPowerMode: ProcessInfo.processInfo.isLowPowerModeEnabled,
```

Comment:
Do we need to track `inLowPowerMode`? If _low power mode_ is on, then `backgroundRefreshStatus` will be disabled?

Reply:
According to Apple, no, even if low power mode is selected, the background task for exposure notification will still run. However, the background task for fetch test results will not, so it may be worthwhile tracking this if we want to let the user know that the test results will not be fetched if in low power mode.

Reply:
Sounds good. Thanks for checking this, guys! 👍😊

Reply:
`setupBackgroundFetchAlert`  is also called by external? Otherwise, we can set it to `private`